desc/vmec.py

```diff
@@ -100,9 +103,29 @@ def load(cls, path, L=-1, M=-1, N=-1, spectral_indexing="fringe"):
         inputs["pressure"] = np.zeros((preset, 2))
         inputs["pressure"][:, 0] = np.arange(0, 2 * preset, 2)
         inputs["pressure"][:, 1] = file.variables["am"][:] * p0
-        inputs["iota"] = np.zeros((preset, 2))
-        inputs["iota"][:, 0] = np.arange(0, 2 * preset, 2)
-        inputs["iota"][:, 1] = file.variables["ai"][:]
+        if profile == "iota":
+            inputs["iota"] = np.zeros((preset, 2))
+            inputs["iota"][:, 0] = np.arange(0, 2 * preset, 2)
+            inputs["iota"][:, 1] = file.variables["ai"][:]
+            inputs["current"] = None
+        if profile == "current":
+            inputs["current"] = np.zeros((preset, 2))
+            inputs["current"][:, 0] = np.arange(0, 2 * preset, 2)
+            inputs["current"][:, 1] = file.variables["ac"][:]
+            # integrate current profile wrt s=rho^2
+            inputs["current"] = np.pad(
+                np.vstack(
+                    (
+                        inputs["current"][:, 0] + 2,
+                        inputs["current"][:, 1] * 2 / (inputs["current"][:, 0] + 2),
+                    )
+                ).T,
+                ((1, 0), (0, 0)),
+            )
+            inputs["current"][:, 1] *= (
+                file.variables["ctor"][:] / (np.sum(inputs["current"][:, 1]) or 1),
```

Comment:
We should also add the "or 1" to line 1321 of `desc/io/input_reader.py`.

Reply:
I already did